src/librustdoc/html/render/print_item.rs

```diff
@@ -73,7 +73,7 @@ pub(super) fn print_item(cx: &Context<'_>, item: &clean::Item, buf: &mut Buffer)
             );
         }
     }
-    write!(buf, "<a class=\"{}\" href=\"\">{}</a>", item.type_(), item.name.as_ref().unwrap());
+    write!(buf, "<span class=\"{}\">{}</span>", item.type_(), item.name.as_ref().unwrap());
```

Comment:
Note that changing this to a `<span>` means it can't be focused, and doesn't participate in the tab sequence. Is that desirable? If we'd like it to stay an `<a>` I think we can do so by putting `href="#"` or `javascript:void(0)`.